app/src/main/res/values/styles_v2.xml

```diff
@@ -649,7 +649,7 @@
   </style>
 
   <style name="Clinic.V2.MaterialButton" parent="Widget.MaterialComponents.Button">
-    <item name="backgroundTint">?attr/colorSecondary</item>
+    <item name="backgroundTint">@color/material_button_background</item>
```

Comment:
This change will apply to all buttons where this style is being used. That is expected, right?

Reply:
Yes, that's expected behaviour. When a button is disabled we want it to be grey and not just keep the actual button color which will confuse the users right?

Reply:
Yup, just wanted to confirm.